backend/src/main/java/com/google/sps/FindMatchQuery.java

```diff
@@ -92,17 +89,17 @@ public Match findMatch(Participant newParticipant) {
       }
       System.out.println("found a match");
       return new Match(
-          newParticipant.getUsername(), currParticipant.getUsername(), duration, currentTimeMillis);
+          newParticipant.getUsername(), currParticipant.getUsername(), duration, clock.millis());
     }
     // No inital match found
     return null;
   }
 
   /** Return true if compatible endTimeAvailable considering duration and padding */
   private boolean isCompatibleTime(
-      long currentTimeMillis, int duration, long newEndTimeAvailable, long currEndTimeAvailable) {
+      int duration, long newEndTimeAvailable, long currEndTimeAvailable) {
```

Comment:
do `new` and `curr` make sense in the context of this method? Maybe something like participant1EndTime, participant2EndTime?

Reply:
changed new to first, curr to second